### PR TITLE
Update DCR Schemas

### PR DESCRIPTION
**Pull request by OllysCoding**
Update the DCR schemas cause they were out of date :(

### CODE REVIEW

File: dotcom-rendering/src/model/front-schema.json [hunk 1]
```diff
@@ -2293,7 +2311,8 @@
                 "PrintShopDesign",
                 "QuizDesign",
                 "RecipeDesign",
-                "ReviewDesign"
+                "ReviewDesign",
+                "VideoDesign"
```
Comment (JamieB-gu) [inline dotcom-rendering/src/model/front-schema.json:2315]: Will we still get the `NewsletterSignup` error if that's not added too?

File: dotcom-rendering/src/model/json-schema.json [hunk 2]
```diff
@@ -2136,7 +2135,8 @@
                 "PrintShopDesign",
                 "QuizDesign",
                 "RecipeDesign",
-                "ReviewDesign"
+                "ReviewDesign",
+                "VideoDesign"
```
Comment (JamieB-gu) [inline dotcom-rendering/src/model/json-schema.json:2139]: Same question as above.

File: dotcom-rendering/src/model/enhanceCollections.ts [hunk 3]
```diff
@@ -101,7 +101,34 @@ const enhanceCards = (
 const decideContainerPalette = (
 	metadata?: { type: FEContainerPalette }[],
 ): DCRContainerPalette | undefined => {
-	return metadata?.length && metadata[0].type ? metadata[0].type : undefined;
+	if (!metadata?.length || metadata[0].type) return undefined;
+	switch (metadata[0].type) {
```
Comment (mxdvl) [inline dotcom-rendering/src/model/enhanceCollections.ts:105]:
```suggestion
	switch (metadata?.[0]?.type) {
```

Comment (OllysCoding) [reply dotcom-rendering/src/model/enhanceCollections.ts:105]: image
Unfortunately you're too smart for typescript!

Comment (OllysCoding) [reply dotcom-rendering/src/model/enhanceCollections.ts]: Done :)

**Review by JamieB-gu — APPROVED** [review]

**Pull request merged**